src/Compilers/VisualBasic/Portable/Syntax/SyntaxKindFacts.vb

```diff
@@ -793,6 +799,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    {"istrue", SyntaxKind.IsTrueKeyword},
                    {"join", SyntaxKind.JoinKeyword},
                    {"key", SyntaxKind.KeyKeyword},
+                   {"load", SyntaxKind.LoadKeyword},
```

Comment:
No, and we should probably remove both "load" and "r" here (per other comment, I don't think they should be considered contextual keywords).